From ee994bb5ffb2aff334e64c1311e4c71b2551e612 Mon Sep 17 00:00:00 2001
From: Zhenhua Luo <zhenhua.luo@freescale.com>
Date: Sun, 1 Dec 2013 17:21:11 -0200
Subject: [PATCH] erlang: skip the host library path check to fix cross-compile

Upstream-Status: Inappropriate [The fix is specific to Yocto build env]

Skip host library path check to fix following Yocto insane check issue:
ERROR: This autoconf log indicates errors, it looked at host include and/or library paths while determining system capabilities.
Rerun configure task after fixing this.

Signed-off-by: Zhenhua Luo <zhenhua.luo@freescale.com>

---
 erts/configure.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/erts/configure.in b/erts/configure.in
index 12dfb9b19b..cd4499d4be 100644
--- a/erts/configure.in
+++ b/erts/configure.in
@@ -1723,10 +1723,10 @@ if test x"$ac_cv_lib_dlpi_dlpi_open" = x"no"; then
 	# ln -s libdlpi.so.1 $try_dlpi_lib/libdlpi.so
 	])
    fi
-   LDFLAGS="-L$try_dlpi_lib -R$try_dlpi_lib $LDFLAGS"
-   unset -v try_dlpi_lib
-   AC_MSG_NOTICE([Extending the search to include /lib])
-   AC_CHECK_LIB(dlpi, dlpi_open)
+#   LDFLAGS="-L$try_dlpi_lib -R$try_dlpi_lib $LDFLAGS"
+#   unset -v try_dlpi_lib
+#   AC_MSG_NOTICE([Extending the search to include /lib])
+#   AC_CHECK_LIB(dlpi, dlpi_open)
    if test x"$ac_cv_lib_dlpi_dlpi_open" = x"no"; then
       LDFLAGS="$save_ldflags"
    fi
